chainermn/communicators/mpi_communicator_base.py

```diff
@@ -1,4 +1,4 @@
-import collections
+from chainer.utils import collections_abc
```

Comment:
Could you move the line to the next to the line `import chainer.utils`?

Reply:
Done, thanks you for the suggestion